testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -500,3 +500,10 @@ def pillar_get(key, minion)
   raise 'File injection failed' unless return_code.zero?
   $server.run("chmod 644 " + dest)
 end
+
+When(/^I migrate "([^"]*)" from OS Salt to the Salt bundle$/) do |host|
+  migrate = "salt #{host} state.apply util.mgr_switch_to_venv_minion"
+  cleanup = %(salt #{host} state.apply util.mgr_switch_to_venv_minion pillar='{"mgr_purge_non_venv_salt_files": True, "mgr_purge_non_venv_salt": True}')
+  $migrate_output = $server.run(migrate, check_errors: false)
+  $cleanup_output = $server.run(cleanup, check_errors: false)
```

Comment:
I notice did you instantiated here two global variables, but then I can't see where we use them 🤔 
In any case, I would first split this step in two, as it is running two different actions: migrate and cleanup.
And second, I would have enabled check_errors, and even maybe have verbose enabled, so we have more log to debug in the report if necessary.

Reply:
To use global variable here is wrong and is a leftover from another approach. I will fix that.
Regarding the cleanup, it is tied to the migration to the Salt bundle according to the documentation. That is why I put it in one step.

![image](https://user-images.githubusercontent.com/12104291/203751695-10c012b1-f90f-4993-8e7d-ac8e9e29e9e0.png)

I had `check_errors` enabled but always got the error that the Minion could not be contacted and the scenario failed. Nevertheless the migration was successful and the Minion was using the Salt bundle. This was the reason why I did not check for errors any more. But we could enable verbosity.

Reply:
I still think we can know better on which action we fail if we have a step for each.
And about "Minion could not be contacted", are you 100% we are not hiding dust below the carpet? Couldn't that be actually a product bug to fix?

Reply:
Ok. Then I will split that up into 2 steps and do some more testing regarding this. If the error/warning is shown again I will post it here.

Reply:
Ok, I did 2 test runs on my local deployment with `check_errors: true` and no errors/warnings so far. I guess this could be a one time issue I had.